varcode/maf.py

```diff
@@ -82,38 +98,60 @@ def load_maf_dataframe(path, nrows=None, verbose=False):
                 df[expected] = df[actual]
                 del df[actual]
             else:
-                raise ValueError("Expected column %s but got %s" % (
-                    expected, actual))
+                error_message = (
+                    "Expected column %s but got %s" % (expected, actual))
+                if raise_on_error:
```

Comment:
This pattern
```
if raise:
  raise ...
else:
  log();
```
is now quite common.
The code could be made cleaner with a helper function in the local scope
```
def raise_or_log(message):
  if (raise_on_error):
    raise ValueError(message)
  else
  logging.warn(message)
```